src/main/drivers/bus_i2c_stm32f10x.c

```diff
@@ -469,6 +469,15 @@ uint16_t i2cGetErrorCounter(void)
     return i2cErrorCount;
 }
 
+static void i2cClockBit(IO_t scl, IO_t sda, bool bit)
+{
+    IOLo(scl); // Set bus scl low
+    delayMicroseconds(10);
+    IOWrite(sda, bit); // Set bus sda high
```

Comment:
Comment is incorrect.

Reply:
Fixed. In fact superseded as I've copied the i2cUnstick() routine from bus_i2c_hal.c for consistency.